src/platform/plugins/shared/data/server/search/strategies/sql_search/sql_search_strategy.ts

```diff
@@ -10,7 +10,7 @@
 import type { IncomingHttpHeaders } from 'http';
 import type { IScopedClusterClient, Logger } from '@kbn/core/server';
 import { catchError, tap } from 'rxjs';
-import type { DiagnosticResult } from '@elastic/transport';
+import type { DiagnosticResult } from '@elastic/elasticsearch';
```

Comment:
this change is because yarn got confused and left the old `@elastic/transport` in the top level and the matching version under `@elastic/elasticsearch/node_modules`.
Since we don't depend directly on `@elastic/transport`, I think it's best to import from the client's library (that re-exports the transport)